Lombiq.Hosting.Tenants.MediaStorageManagement/Startup.cs

```diff
@@ -18,11 +18,13 @@ public class Startup : StartupBase
 
     public override void ConfigureServices(IServiceCollection services)
     {
-        var maximumStorageQuota =
+        var maximumStorageQuotaBytes =
+            _shellConfiguration.GetValue<long?>(
+                "Lombiq_Hosting_Tenants_MediaStorageManagement:Media_Storage_Management_Options:MaximumSpaceBytes") ??
             _shellConfiguration.GetValue<long?>(
                 "Lombiq_Hosting_Tenants_MediaStorageManagement:Media_Storage_Management_Options:MaximumSpace");
         services.Configure<MediaStorageManagementOptions>(options =>
-            options.MaximumStorageQuota = maximumStorageQuota ?? MaximumStorageQuota);
+            options.MaximumStorageQuotaBytes = maximumStorageQuotaBytes ?? MaximumStorageQuotaBytes);
```

Comment:
I just noticed that the option is actually called StorageQuota, while the config key Size... That's bad. Please fix this by renaming the latter to use the StorageQuota terminology (and all "space" identifiers too).

Furthermore, the key has wrong underscores, so it should be `"Lombiq_Hosting_Tenants_MediaStorageManagement:MediaStorageManagementOptions:MaximumStorageQuotaBytes"`. But! We don't actually need this indirection since we won't have a lot of layers for config here, so rather name it `"Lombiq_Hosting_Tenants_MediaStorageManagement:MaximumStorageQuotaBytes"`.

Reply:
> while the config key Size

Where is this config key? I can't find it.

> But! We don't actually need this indirection since we won't have a lot of layers for config here, so rather name it "Lombiq_Hosting_Tenants_MediaStorageManagement:MaximumStorageQuotaBytes".

Then I need to delete `MediaStorageManagementOptions.cs` no?

Reply:
I mean, the property in `MediaStorageManagementOptions` uses different terminology than the configuration key read above. I'm asking you to please rename the configuration key to `"Lombiq_Hosting_Tenants_MediaStorageManagement:MaximumStorageQuotaBytes"` (so it matches what's in the options class), and adapt the terminology used by the other identifiers (variables, properties) too.

Reply:
Ah I guess you wanted to write Space but you wrote

> Size

That's why I was confised.